### DISCUSSION TITLE
bnn-pynq example

### DISCUSSION THREAD

**Discussion by btwbtw01** [Xilinx/finn#539] [Q&A]
Hi, everyone 
I'm running tfc_end2end_example. It shows the error when building the hardware.
![image](https://user-images.githubusercontent.com/85877169/153199666-f25a6c46-0ae4-42cd-abf4-b5f561f8e969.png)
Have anybody met this error before?

**Comment by btwbtw01** [comment]
Sorry the picture isn't showing the error. I post a new one
![image](https://user-images.githubusercontent.com/85877169/153200889-9a93dbd3-2349-429a-808c-20049d088cfb.png)

**Comment by sakshigoel155** [comment]
Hi,

Maybe it is the same issue mentioned here. https://github.com/Xilinx/finn/discussions/500
Otherwise you can check further logs based on same discussion
